docs/api.md

```diff
@@ -1969,6 +1986,20 @@ requested by the specified OAuth client.
 <!--end-route-post-accountscoped-key-data-->
 
 
+#### POST /oauth/authorization
+
+:lock: HAWK-authenticated with session token
+<!--begin-route-post-oauthauthorization-->
+Authorize a new OAuth client connection to the user's account,
+returning a short-lived authentication code that the client can
+exchange for access tokens at the OAuth token endpoint.
+
+This route behaves like the (oauth-server /authorization endpoint)[../fxa-oauth-server/docs/api.md#post-v1authorization]
+except that it is authenticated directly with a sessionToken
+rather than with a BrowserID assertion.
+<!--end-route-post-oauthauthorization-->
+
```

Comment:
Humph, the auto-generated API docs can't see into the request and response schemas for this route, because they't defined (in the backend service)[https://github.com/mozilla/fxa-auth-server/pull/2932/files#diff-ed3e84d4d5bc6cbc06e0c9166915cd07R15] rather than directly on the route.

I could try to figure out how to parse them out of there, but I also heard some talk about trying to move towards OpenAPI for specifying and documenting all our routes, so I'm not sure where my effort is best directed here. @shane-tomlinson thoughts?

Reply:
Not directed at me I realise, but answering as I wrote it, I think at this point we can consider the auto-generated docs script a failed experiment. Maybe we should revert to maintaining this document by hand and disabling the script.

Reply:
> I think at this point we can consider the auto-generated docs script a failed experiment.

Well, it was a fun experiment while it lasted :-)

Reply:
I went ahead and added some basic docs by hand.

Reply:
Ref: #2988.